### DISCUSSION TITLE
Question about register 20880 to app instance

### DISCUSSION THREAD

**Discussion by yx-Yaoxaing** [apache/dubbo#13888] [Question] [answered]
![image](https://github.com/apache/dubbo/assets/56578625/2fbabe2a-8c26-4782-946e-616f4b57d698)
![image](https://github.com/apache/dubbo/assets/56578625/b6d61d0c-d147-4bdc-838b-5f7d92d5a51a)

服务dubbo配置
dubbo:
  registry:
    address: nacos://${nacos.server.addr}
    parameters:
      namespace: ${discovery.namespace}
    username: ${nacos.username}
    password: ${nacos.password}
  protocol:
    name: dubbo
    port: -1
  consumer:
    timeout: 1500
    retries: 2
    reconnect: false
    check: false
    filter: dubboConusmerFilter
  application:
    qos-enable: false

**Comment by AlbumenJ** [comment, marked as answer]
This is work as expected and is registered by application level service discovery.
